src/Microsoft.AspNet.OData.Shared/Formatter/EdmLibHelpers.cs

```diff
@@ -654,6 +657,19 @@ public static IEdmType GetElementType(IEdmTypeReference edmTypeReference)
                         return;
                     }
 
+                    if (segment is OperationSegment operationSegment)
+                    {
+                        IEdmFunction function = operationSegment.Operations.SingleOrDefault() as IEdmFunction;
```

Comment:
Shouldn't be limited to functions

Reply:
I've fixed this locally, but I'll hold off pushing a new iteration until we've resolved my previous question re: `GetHashCode`